modules/julia/README.md

```diff
@@ -0,0 +1,63 @@
+OpenCV Julia Bindings
+============================
+This module contains some limited functionality that allows OpenCV functions be used from Julia. Upon installation the binding files are automatically registered with Julia's package manager like any normal package.
+
+This module requires Julia 1.4 and the CxxWrap.jl 0.10.
+
+Build
+-----
+The Julia module is fully integrated into the OpenCV build system. If cmake finds a Julia executable available on the host system while configuring OpenCV, it will attempt to generate Julia wrappers for all OpenCV modules. If cmake is having trouble finding your Julia installation, you can explicitly point it to the Julia executable by defining the `Julia_EXECUTABLE` variable. For example:
+
+    cmake -DJulia_EXECUTABLE=/home/user/julia-1.4.1/bin ..
+
+If you prefer using the gui version of cmake (cmake-gui), you can use the *Add Entry* option in the GUI to manually add the *path* variable `Julia_EXECUTABLE`.
+
+
+Install
+-------
+By default the Julia package is installed in `CMAKE_CURRENT_BINARY_DIR`, you can change this by setting the `JULIA_PKG_INSTALL_PATH` cmake variable. The package is automatically registered with the Julia package manager.
+
```

Comment:
it should include the instructions how to install CxxWrap prior to compiling OpenCV.

Reply:
CxxWrap from the julia package manager should now work with this. Also, the CxxWrap page itself has detailed installation instructions. Should I instead link to that and just add instructions about how to use the package manager version?

Reply:
installing CxxWrap is just 2 commands. please, put these instructions inside this document

Reply:
Done